repose-aggregator/components/filters/uri-stripper-filter/src/main/scala/org/openrepose/filters/uristripper/UriStripperFilter.scala

```diff
@@ -135,9 +162,47 @@ class UriStripperFilter @Inject()(configurationService: ConfigurationService)
     }
   }
 
+  private def handleJsonRequestLinks(wrappedRequest: HttpServletRequestWrapper, previousToken: Option[String], nextToken: Option[String], token: Option[String], applicableLinkPaths: List[LinkPath]): Try[HttpServletRequestWrapper] = {
+    val tryParsedJson = Try(Json.parse(wrappedRequest.getInputStream))
+    applicableLinkPaths.foldLeft(tryParsedJson)(transformJsonLink(_, _, token.get, previousToken, nextToken, isRequest = true)) match {
```

Comment:
Are we sure we aren't going to want them for the contextual removal on the request body like is used for the contextual insertion on the response body?

If we don't, then there is a lot of this that can be cleaned up and maybe even simplified.